chef_master/source/release_notes.rst

```diff
@@ -2985,6 +2985,13 @@ The launchd resource has the following actions:
 ``:restart``
    Restart a launchd managed daemon or agent.
 
+``:nothing``
+   .. tag resources_common_actions_nothing
+
+   Define this resource block to do nothing until notified by another resource to take action. When this resource is notified, this resource block is either run immediately or it is queued up to be run at the end of the Chef Client run.
```

Comment:
s/until/unless ?

Reply:
Consider: This resource block does not act unless notified by another resource to take action. Once notified, this resource block either runs immediately or is queued up to run at the end of the Chef Client run.

Reply:
Updated